taotie/sources/arxiv.py

```diff
@@ -20,7 +20,8 @@ class Arxiv(BaseSource):
 
     def __init__(self, sink: MessageQueue, verbose: bool = False, **kwargs):
         BaseSource.__init__(self, sink=sink, verbose=verbose, **kwargs)
-        self.authors = os.environ.get("ARXIV_AUTHORS", "").split(",")
+        with open('arxiv_author.json') as f:
+            self.authors = json.load(f)
```

Comment:
'arxiv_author.json' contains a map where values are lists. We would need to merge all the lists into one authors list.